completions/bash/podman

```diff
@@ -982,6 +982,7 @@ _podman_network_create() {
     --subnet
      "
     local boolean_options="
+    --disabledns
```

Comment:
looks like you need another space or two here, or maybe a tab replacement

Reply:
i dont understand, none of the other options have indentation.